src/blocks/scratch3_looks.js

```diff
@@ -119,7 +119,7 @@ class Scratch3LooksBlocks {
     _positionBubble (target) {
         const bubbleState = this._getBubbleState(target);
         const [bubbleWidth, bubbleHeight] = this.runtime.renderer.getSkinSize(bubbleState.drawableId);
-        const targetBounds = target.getBounds();
+        const targetBounds = target.getBoundsForBubble();
```

Comment:
I was going to suggest adding a `null` check in this call, since `target.getBoundsForBubble()` can return `null`, but then I noticed two things:
- `target.getBounds()` could return `null` in the same situation (no renderer) so this change isn't making things any worse, but
- the line above this calls a method on `this.runtime.renderer` without checking first, which means that the check in `target.getBoundsForBubble()` is currently unnecessary.

I guess I would low-key advocate for removing the check in `getBoundsForBubble()` but I'd also be OK with leaving this as-is. At some point I'd like to make a "discipline" pass where we decide which functions can or can't assume the presence of a renderer, audio engine, storage module, etc., if for no other reason than developer ergonomics.